SmartTV-AGH.txt

```diff
@@ -0,0 +1,250 @@
+[Adblock Plus 3.6]
+! Title: Perflyst and Dandelion Sprout's Smart-TV Blocklist for AdGuard Home
+! Version: 11January2020v3-Beta
```

Comment:
I'm normally keen on having a v[number] in at least my own lists' versions, in case one would make multiple edits to a list on the same day. The `-Beta` can however be omitted, as I had simply added it to show to users that the list hadn't been 110% waterproofed just yet.